### ISSUE TITLE
Optimise for SEO

### ISSUE THREAD

**Issue by j4-m** [localgovdrupal/docs#99]
Presently the generated metatags as possibly a bit lacking, e.g. for the meta title\description tag for services documentation for content designers is: 

```
<title>Services | LocalGov Drupal Docs</title>
```

```
<meta name="description" content="LocalGov Drupal is an open source collaboration between UK councils and Drupal developers.">
```
I'd imagine this could be optimised somewhat. Any thoughts @MichaelDCR @FinnERLewis

**Comment by MichaelDCR** [comment]
yes - thanks @j4-m I hadn't even considered being able to add meta tags 

I'll have a look, ask questions if I can't see how to do it, and how best to do the workflow. 

Adding @msayoung
